src/AppBundle/Form/UserAddressType.php

```diff
@@ -42,6 +42,4 @@ public function getBlockPrefix()
     {
         return 'appbundle_user';
     }
-
```

Comment:
comment for above, replace this:

$resolver->setDefaults([
             'data_class' => 'AppBundle\Entity\User'
         ]);

to this:

$resolver->setDefaults([
             'data_class' => User::class,
         ]);